src/ucp/core/ucp_ep.h

```diff
@@ -162,8 +162,10 @@ enum {
     UCP_EP_INIT_CREATE_AM_LANE_ONLY    = UCS_BIT(8),  /**< Endpoint requires an AM lane only */
     UCP_EP_INIT_ALLOW_KA_AUX_TL        = UCS_BIT(9),  /**< Endpoint allows selecting of auxiliary
                                                            transports for keepalive lane */
-    UCP_EP_INIT_ALLOW_AM_AUX_TL        = UCS_BIT(10)  /**< Endpoint allows selecting of auxiliary
+    UCP_EP_INIT_ALLOW_AM_AUX_TL        = UCS_BIT(10), /**< Endpoint allows selecting of auxiliary
                                                            transports for AM lane */
+    UCP_EP_INIT_KA_FROM_EXIST_LANES    = UCS_BIT(11)  /**< Use only existing lanes to create
```

Comment:
now it conflicts with `UCP_EP_INIT_ALLOW_KA_AUX_TL`.
Is it possible to use `tl_bitmap`  from `ucp_wireup_select_context_t`  instead?

Reply:
> now it conflicts with UCP_EP_INIT_ALLOW_KA_AUX_TL.

yes, this is a problem

> now it conflicts with UCP_EP_INIT_ALLOW_KA_AUX_TL.
Is it possible to use tl_bitmap from ucp_wireup_select_context_t instead?

how it should be done? we should have a control to define whether we should transports from existing lanes to create KA lane or not

Reply:
discussed offline with @brminich that we coudl remove `UCP_EP_INIT_ALLOW_KA_AUX_TL` flag - thanks, good catch, fixed.